### PR TITLE
Update jackson to 2.13.2, databind to 2.13.2.1 - CVE-2020-36518

### PR DESCRIPTION
**Pull request by lmolkova**
See https://github.com/FasterXML/jackson-databind/issues/2816

### CODE REVIEW

File: eng/versioning/external_dependencies.txt [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
-com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.13.1
-com.fasterxml.jackson.datatype:jackson-datatype-jdk8;2.13.1
-com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.13.1
-com.fasterxml.jackson.module:jackson-module-parameter-names;2.13.1
-com.fasterxml.jackson.module:jackson-module-scala_2.12;2.13.1
+com.fasterxml.jackson.core:jackson-annotations;2.13.2
+com.fasterxml.jackson.core:jackson-core;2.13.2
+com.fasterxml.jackson.core:jackson-databind;2.13.2
+com.fasterxml.jackson.dataformat:jackson-dataformat-avro;2.13.2
+com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.13.2
```
Comment (FabianMeiswinkel) [inline eng/versioning/external_dependencies.txt:14]: Please move afterburner module to 2.13.2 as well
cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;2.13.2

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM - except for the afterburner module that needs to be moved to 2.13.2 as well

**Review by kasobol-msft — APPROVED** [review]

File: common/perf-test-core/pom.xml [hunk 2]
```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.13.1</version>  <!-- {x-version-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
+      <version>2.13.2.1</version>  <!-- {x-version-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
```
Comment (mssfang) [inline common/perf-test-core/pom.xml:67]: is 2.13.2?

**Review by mssfang — APPROVED** [review]

File: common/perf-test-core/pom.xml [hunk 3]
```diff
@@ -46,7 +46,7 @@
           <rules>
             <bannedDependencies>
               <includes combine.children="append">
-                <include>com.fasterxml.jackson.core:jackson-databind:[2.13.1]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
+                <include>com.fasterxml.jackson.core:jackson-databind:[2.13.2.1]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
```
Comment (joshfree) [inline common/perf-test-core/pom.xml:49]: should this be 2.13.2 and not 2.13.2.1 ? 

https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind

**Review by joshfree — APPROVED** [review]

**Review by ki1729 — APPROVED** [review]
:shipit:

File: common/perf-test-core/pom.xml (hunk 2 above)
Comment (lmolkova) [reply common/perf-test-core/pom.xml:67]: jackson-databind `2.13.2.1` is actually the one with the fix

**Pull request merged**